conans/client/conan_api.py

```diff
@@ -338,26 +337,29 @@ def get_test_conanfile_path(tf):
         test_conanfile_path = get_test_conanfile_path(test_folder)
         self._recorder.add_recipe_being_developed(reference)
 
+        ret = None
         if test_conanfile_path:
             pt = PackageTester(self._manager, self._user_io)
-            pt.install_build_and_test(test_conanfile_path, reference, profile,
-                                      remote, update, build_modes=build_modes,
-                                      manifest_folder=manifest_folder,
-                                      manifest_verify=manifest_verify,
-                                      manifest_interactive=manifest_interactive,
-                                      keep_build=keep_build,
-                                      test_build_folder=test_build_folder)
+            ret = pt.install_build_and_test(test_conanfile_path, reference, profile,
+                                            remote, update, build_modes=build_modes,
+                                            manifest_folder=manifest_folder,
+                                            manifest_verify=manifest_verify,
+                                            manifest_interactive=manifest_interactive,
+                                            keep_build=keep_build,
+                                            test_build_folder=test_build_folder)
         else:
-            self._manager.install(reference=reference,
-                                  install_folder=None,  # Not output anything
-                                  manifest_folder=manifest_folder,
-                                  manifest_verify=manifest_verify,
-                                  manifest_interactive=manifest_interactive,
-                                  remote_name=remote,
-                                  profile=profile,
-                                  build_modes=build_modes,
-                                  update=update,
-                                  keep_build=keep_build)
+            ret = self._manager.install(reference=reference,
```

Comment:
This is weird. What is returning the manager and the package tester? I think we don't want these returns, just always the recorder.get_info(). Right?